packages/api/src/external/hie/__tests__/register-facility.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { faker } from "@faker-js/faker";
+import { USState } from "@metriport/api-sdk";
+import { OrgType } from "@metriport/core/domain/organization";
+
+export const getCxOrganizationNameAndOidResult = {
+  name: faker.company.name(),
+  oid: faker.string.uuid(),
+  type: OrgType.acuteCare,
+};
+
+export const addressWithCoordinates = {
```

Comment:
nit: when we create the object manually, it's useful to have the type we're trying to represent w/ these props.

Here and above.